jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/CachedContentFactory.java

```diff
@@ -67,7 +66,7 @@ public class CachedContentFactory implements HttpContent.ContentFactory
      * Constructor.
      *
      * @param parent the parent resource cache
-     * @param factory the resource factory
+     * @param factory the resource
```

Comment:
strange edit, this param is a ResourceFactory.

Reply:
It's probably a leftover of the many refactorings I made around this class. Let me revert this change.